Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableColorTheme.cs

```diff
@@ -323,6 +353,23 @@ protected bool TrySetRendererColor(Color colour, ThemeStateProperty property, in
             return true;
         }
 
+        /// <summary>
+        /// Try to set color on Image
+        /// If false, no Image was found.
+        /// </summary>
+        protected bool TrySetUnityUiImageColor(Color colour, ThemeStateProperty property, int index, float percentage)
+        {
+            Image img = Host.GetComponent<Image>();
```

Comment:
>Image [](start = 12, length = 5)

Instead of Image, you should use Graphic (Image's base class).

Reply:
i swapped out all "Image" references with Graphic